src/screens/Learning.js

```diff
@@ -60,8 +60,20 @@ export default ({
 
   const setAnswerOptionsCallback = (original, current) => {
     const originWithoutCurrent = original.filter(phr => phr.id !== current.id);
-    const randomFromAll = shuffleArray(originWithoutCurrent).slice(0, 3);
-    const randomWithCorrect = shuffleArray([...randomFromAll, current]);
+
+    const phraseCategoryId = categories.find(cat =>
+      cat.phrasesIds.includes(current?.id),
+    );
+
+    const allNewPhrases = getPhrasesForCategoryId(phraseCategoryId?.id);
```

Comment:
As discussed when finished with appendix 2 this will have to be refactored and moved to thunk action